include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -292,7 +292,10 @@ typedef struct _sgx_secs
     uint8_t reserved3[96];  /* 160 */
     uint16_t isvvprodid;    /* 256 */
     uint16_t isvsvn;        /* 258 */
-    uint8_t reserved[3836]; /* 260 */
+    uint8_t reserved4[20];  /* 260 */
```

Comment:
Field offsets and sizes have to be identical to secs_t defined in [arch.h](https://github.com/intel/linux-sgx/blob/d3bd1571240bcdf85734c232a4f0c86828443ebb/common/inc/internal/arch.h#L59).